<?xml version='1.0' encoding='UTF-8'?>
<testsuite hostname="knoldus-Vostro-3546" name="com.knoldus.TwitterSpec" tests="6" errors="0" failures="6" skipped="0" time="2.43">
                     <properties>
      <property name="jline.esc.timeout" value="0"/><property name="jna.platform.library.path" value="/usr/lib/x86_64-linux-gnu:/lib/x86_64-linux-gnu:/lib64:/usr/lib:/lib"/><property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/><property name="sun.boot.library.path" value="/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/amd64"/><property name="java.vm.version" value="25.151-b12"/><property name="java.vm.vendor" value="Oracle Corporation"/><property name="java.vendor.url" value="http://java.oracle.com/"/><property name="path.separator" value=":"/><property name="jna.loaded" value="true"/><property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/><property name="file.encoding.pkg" value="sun.io"/><property name="user.country" value="IN"/><property name="sun.java.launcher" value="SUN_STANDARD"/><property name="sun.os.patch.level" value="unknown"/><property name="jna.nosys" value="true"/><property name="java.vm.specification.name" value="Java Virtual Machine Specification"/><property name="user.dir" value="/home/knoldus/Documents/assignments/kip-assignment/26-07-2018/twitter-analyse-app"/><property name="java.runtime.version" value="1.8.0_151-b12"/><property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/><property name="java.endorsed.dirs" value="/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/endorsed"/><property name="os.arch" value="amd64"/><property name="java.io.tmpdir" value="/tmp"/><property name="line.separator" value="
"/><property name="java.vm.specification.vendor" value="Oracle Corporation"/><property name="os.name" value="Linux"/><property name="sun.jnu.encoding" value="UTF-8"/><property name="jnidispatch.path" value="/tmp/jna--566319230/jna8778053371861520414.tmp"/><property name="java.library.path" value="/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/><property name="java.specification.name" value="Java Platform API Specification"/><property name="java.class.version" value="52.0"/><property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/><property name="os.version" value="4.15.0-29-generic"/><property name="user.home" value="/home/knoldus"/><property name="user.timezone" value="Asia/Kolkata"/><property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/><property name="file.encoding" value="UTF-8"/><property name="java.specification.version" value="1.8"/><property name="java.class.path" value="/usr/share/sbt/bin/sbt-launch.jar"/><property name="user.name" value="knoldus"/><property name="jline.shutdownhook" value="false"/><property name="java.vm.specification.version" value="1.8"/><property name="sun.java.command" value="/usr/share/sbt/bin/sbt-launch.jar test"/><property name="java.home" value="/home/knoldus/Documents/application/jdk1.8.0_151/jre"/><property name="sun.arch.data.model" value="64"/><property name="user.language" value="en"/><property name="java.specification.vendor" value="Oracle Corporation"/><property name="awt.toolkit" value="sun.awt.X11.XToolkit"/><property name="java.vm.info" value="mixed mode"/><property name="java.version" value="1.8.0_151"/><property name="java.ext.dirs" value="/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/ext:/usr/java/packages/lib/ext"/><property name="sun.boot.class.path" value="/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/resources.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/rt.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/sunrsasign.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/jsse.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/jce.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/charsets.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/lib/jfr.jar:/home/knoldus/Documents/application/jdk1.8.0_151/jre/classes"/><property name="java.vendor" value="Oracle Corporation"/><property name="file.separator" value="/"/><property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/><property name="sun.io.unicode.encoding" value="UnicodeLittle"/><property name="sun.cpu.endian" value="little"/><property name="sun.desktop" value="gnome"/><property name="sun.cpu.isalist" value=""/>
    </properties>
                     <testcase classname="com.knoldus.TwitterSpec" name="Twitter class should fetch the tweets" time="1.065">
                                                 <failure message="401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32
" type="twitter4j.TwitterException">401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32

Relevant discussions can be found on the Internet at:
	http://www.google.co.jp/search?q=506c3b98 or
	http://www.google.co.jp/search?q=11ed9ae8
TwitterException{exceptionCode=[506c3b98-11ed9ae8], statusCode=401, message=Could not authenticate you., code=32, retryAfter=-1, rateLimitStatus=null, version=4.0.6}
	at twitter4j.HttpClientImpl.handleRequest(HttpClientImpl.java:164)
	at twitter4j.HttpClientBase.request(HttpClientBase.java:57)
	at twitter4j.HttpClientBase.get(HttpClientBase.java:75)
	at twitter4j.TwitterImpl.get(TwitterImpl.java:1786)
	at twitter4j.TwitterImpl.search(TwitterImpl.java:268)
	at com.knoldus.Twitter.retrieveTweet(Twitter.scala:29)
	at com.knoldus.Twitter.$anonfun$getNumberOfTweets$1(Twitter.scala:35)
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:654)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
</failure>
                                               </testcase><testcase classname="com.knoldus.TwitterSpec" name="Twitter class should fetch the average number of tweets corresponding to input" time="0.264">
                                                 <failure message="401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32
" type="twitter4j.TwitterException">401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32

Relevant discussions can be found on the Internet at:
	http://www.google.co.jp/search?q=506c3b98 or
	http://www.google.co.jp/search?q=11ed9ae8
TwitterException{exceptionCode=[506c3b98-11ed9ae8], statusCode=401, message=Could not authenticate you., code=32, retryAfter=-1, rateLimitStatus=null, version=4.0.6}
	at twitter4j.HttpClientImpl.handleRequest(HttpClientImpl.java:164)
	at twitter4j.HttpClientBase.request(HttpClientBase.java:57)
	at twitter4j.HttpClientBase.get(HttpClientBase.java:75)
	at twitter4j.TwitterImpl.get(TwitterImpl.java:1786)
	at twitter4j.TwitterImpl.search(TwitterImpl.java:268)
	at com.knoldus.Twitter.retrieveTweet(Twitter.scala:29)
	at com.knoldus.Twitter.$anonfun$getNumberOfTweets$1(Twitter.scala:35)
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:654)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
</failure>
                                               </testcase><testcase classname="com.knoldus.TwitterSpec" name="Twitter class should fetch the average number of likes per tweet corresponding to input" time="0.285">
                                                 <failure message="401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32
" type="twitter4j.TwitterException">401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32

Relevant discussions can be found on the Internet at:
	http://www.google.co.jp/search?q=506c3b98 or
	http://www.google.co.jp/search?q=11ed9ae8
TwitterException{exceptionCode=[506c3b98-11ed9ae8], statusCode=401, message=Could not authenticate you., code=32, retryAfter=-1, rateLimitStatus=null, version=4.0.6}
	at twitter4j.HttpClientImpl.handleRequest(HttpClientImpl.java:164)
	at twitter4j.HttpClientBase.request(HttpClientBase.java:57)
	at twitter4j.HttpClientBase.get(HttpClientBase.java:75)
	at twitter4j.TwitterImpl.get(TwitterImpl.java:1786)
	at twitter4j.TwitterImpl.search(TwitterImpl.java:268)
	at com.knoldus.Twitter.retrieveTweet(Twitter.scala:29)
	at com.knoldus.Twitter.$anonfun$getAverageLikes$1(Twitter.scala:44)
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:654)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
</failure>
                                               </testcase><testcase classname="com.knoldus.TwitterSpec" name="Twitter class should give exception when there exists no tweet for the input string" time="0.276">
                                                 <failure message="Expected exception java.lang.ArithmeticException to be thrown, but twitter4j.TwitterException was thrown" type="org.scalatest.exceptions.TestFailedException">org.scalatest.exceptions.TestFailedException: Expected exception java.lang.ArithmeticException to be thrown, but twitter4j.TwitterException was thrown
	at org.scalatest.RecoverMethods.newAssertionFailedExceptionForRecover(RecoverMethods.scala:206)
	at org.scalatest.RecoverMethods.newAssertionFailedExceptionForRecover$(RecoverMethods.scala:205)
	at org.scalatest.AsyncFlatSpec.newAssertionFailedExceptionForRecover(AsyncFlatSpec.scala:2219)
	at org.scalatest.RecoverMethods.$anonfun$recoverToSucceededIf$1(RecoverMethods.scala:194)
	at scala.concurrent.Future.$anonfun$transform$2(Future.scala:240)
	at scala.concurrent.Future.$anonfun$transform$1(Future.scala:240)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at org.scalatest.concurrent.SerialExecutionContext.recRunNow(SerialExecutionContext.scala:120)
	at org.scalatest.concurrent.SerialExecutionContext.runNow(SerialExecutionContext.scala:111)
	at org.scalatest.AsyncSuperEngine.runTestImpl(AsyncEngine.scala:295)
	at org.scalatest.AsyncFlatSpecLike.runTest(AsyncFlatSpecLike.scala:1709)
	at org.scalatest.AsyncFlatSpecLike.runTest$(AsyncFlatSpecLike.scala:1689)
	at org.scalatest.AsyncFlatSpec.runTest(AsyncFlatSpec.scala:2219)
	at org.scalatest.AsyncFlatSpecLike.$anonfun$runTests$1(AsyncFlatSpecLike.scala:1767)
	at org.scalatest.AsyncSuperEngine.$anonfun$runTestsInBranch$2(AsyncEngine.scala:437)
	at org.scalatest.Status.$anonfun$thenRun$1(Status.scala:225)
	at org.scalatest.Status.$anonfun$thenRun$1$adapted(Status.scala:223)
	at org.scalatest.ScalaTestStatefulStatus.whenCompleted(Status.scala:607)
	at org.scalatest.Status.thenRun(Status.scala:223)
	at org.scalatest.Status.thenRun$(Status.scala:221)
	at org.scalatest.ScalaTestStatefulStatus.thenRun(Status.scala:503)
	at org.scalatest.AsyncSuperEngine.$anonfun$runTestsInBranch$1(AsyncEngine.scala:437)
	at scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
	at scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
	at scala.collection.immutable.List.foldLeft(List.scala:86)
	at org.scalatest.AsyncSuperEngine.traverseSubNodes$1(AsyncEngine.scala:419)
	at org.scalatest.AsyncSuperEngine.runTestsInBranch(AsyncEngine.scala:475)
	at org.scalatest.AsyncSuperEngine.$anonfun$runTestsInBranch$1(AsyncEngine.scala:460)
	at scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
	at scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
	at scala.collection.immutable.List.foldLeft(List.scala:86)
	at org.scalatest.AsyncSuperEngine.traverseSubNodes$1(AsyncEngine.scala:419)
	at org.scalatest.AsyncSuperEngine.runTestsInBranch(AsyncEngine.scala:483)
	at org.scalatest.AsyncSuperEngine.runTestsImpl(AsyncEngine.scala:550)
	at org.scalatest.AsyncFlatSpecLike.runTests(AsyncFlatSpecLike.scala:1767)
	at org.scalatest.AsyncFlatSpecLike.runTests$(AsyncFlatSpecLike.scala:1766)
	at org.scalatest.AsyncFlatSpec.runTests(AsyncFlatSpec.scala:2219)
	at org.scalatest.Suite.run(Suite.scala:1147)
	at org.scalatest.Suite.run$(Suite.scala:1129)
	at org.scalatest.AsyncFlatSpec.org$scalatest$AsyncFlatSpecLike$$super$run(AsyncFlatSpec.scala:2219)
	at org.scalatest.AsyncFlatSpecLike.$anonfun$run$1(AsyncFlatSpecLike.scala:1812)
	at org.scalatest.AsyncSuperEngine.runImpl(AsyncEngine.scala:620)
	at org.scalatest.AsyncFlatSpecLike.run(AsyncFlatSpecLike.scala:1812)
	at org.scalatest.AsyncFlatSpecLike.run$(AsyncFlatSpecLike.scala:1810)
	at org.scalatest.AsyncFlatSpec.run(AsyncFlatSpec.scala:2219)
	at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:314)
	at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:507)
	at sbt.TestRunner.runTest$1(TestFramework.scala:106)
	at sbt.TestRunner.run(TestFramework.scala:117)
	at sbt.TestFramework$$anon$2$$anonfun$$lessinit$greater$1.$anonfun$apply$1(TestFramework.scala:262)
	at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:233)
	at sbt.TestFramework$$anon$2$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:262)
	at sbt.TestFramework$$anon$2$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:262)
	at sbt.TestFunction.apply(TestFramework.scala:271)
	at sbt.Tests$.$anonfun$toTask$1(Tests.scala:281)
	at sbt.std.Transform$$anon$3.$anonfun$apply$2(System.scala:46)
	at sbt.std.Transform$$anon$4.work(System.scala:66)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:263)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
	at sbt.Execute.work(Execute.scala:272)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:263)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:174)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: 401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32

Relevant discussions can be found on the Internet at:
	http://www.google.co.jp/search?q=506c3b98 or
	http://www.google.co.jp/search?q=11ed9ae8
TwitterException{exceptionCode=[506c3b98-11ed9ae8], statusCode=401, message=Could not authenticate you., code=32, retryAfter=-1, rateLimitStatus=null, version=4.0.6}
	at twitter4j.HttpClientImpl.handleRequest(HttpClientImpl.java:164)
	at twitter4j.HttpClientBase.request(HttpClientBase.java:57)
	at twitter4j.HttpClientBase.get(HttpClientBase.java:75)
	at twitter4j.TwitterImpl.get(TwitterImpl.java:1786)
	at twitter4j.TwitterImpl.search(TwitterImpl.java:268)
	at com.knoldus.Twitter.retrieveTweet(Twitter.scala:29)
	at com.knoldus.Twitter.$anonfun$getAverageLikes$1(Twitter.scala:44)
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:654)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
</failure>
                                               </testcase><testcase classname="com.knoldus.TwitterSpec" name="Twitter class should fetch the average number of re-tweets per tweet corresponding to inputwhile computing average likes per tweet" time="0.283">
                                                 <failure message="401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32
" type="twitter4j.TwitterException">401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32

Relevant discussions can be found on the Internet at:
	http://www.google.co.jp/search?q=506c3b98 or
	http://www.google.co.jp/search?q=11ed9ae8
TwitterException{exceptionCode=[506c3b98-11ed9ae8], statusCode=401, message=Could not authenticate you., code=32, retryAfter=-1, rateLimitStatus=null, version=4.0.6}
	at twitter4j.HttpClientImpl.handleRequest(HttpClientImpl.java:164)
	at twitter4j.HttpClientBase.request(HttpClientBase.java:57)
	at twitter4j.HttpClientBase.get(HttpClientBase.java:75)
	at twitter4j.TwitterImpl.get(TwitterImpl.java:1786)
	at twitter4j.TwitterImpl.search(TwitterImpl.java:268)
	at com.knoldus.Twitter.retrieveTweet(Twitter.scala:29)
	at com.knoldus.Twitter.$anonfun$getAverageReTweets$1(Twitter.scala:55)
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:654)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
</failure>
                                               </testcase><testcase classname="com.knoldus.TwitterSpec" name="Twitter class should give exception when there exists no tweet for the input string while computing average re-tweet per tweet" time="0.257">
                                                 <failure message="Expected exception java.lang.ArithmeticException to be thrown, but twitter4j.TwitterException was thrown" type="org.scalatest.exceptions.TestFailedException">org.scalatest.exceptions.TestFailedException: Expected exception java.lang.ArithmeticException to be thrown, but twitter4j.TwitterException was thrown
	at org.scalatest.RecoverMethods.newAssertionFailedExceptionForRecover(RecoverMethods.scala:206)
	at org.scalatest.RecoverMethods.newAssertionFailedExceptionForRecover$(RecoverMethods.scala:205)
	at org.scalatest.AsyncFlatSpec.newAssertionFailedExceptionForRecover(AsyncFlatSpec.scala:2219)
	at org.scalatest.RecoverMethods.$anonfun$recoverToSucceededIf$1(RecoverMethods.scala:194)
	at scala.concurrent.Future.$anonfun$transform$2(Future.scala:240)
	at scala.concurrent.Future.$anonfun$transform$1(Future.scala:240)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at org.scalatest.concurrent.SerialExecutionContext.recRunNow(SerialExecutionContext.scala:120)
	at org.scalatest.concurrent.SerialExecutionContext.runNow(SerialExecutionContext.scala:111)
	at org.scalatest.AsyncSuperEngine.runTestImpl(AsyncEngine.scala:295)
	at org.scalatest.AsyncFlatSpecLike.runTest(AsyncFlatSpecLike.scala:1709)
	at org.scalatest.AsyncFlatSpecLike.runTest$(AsyncFlatSpecLike.scala:1689)
	at org.scalatest.AsyncFlatSpec.runTest(AsyncFlatSpec.scala:2219)
	at org.scalatest.AsyncFlatSpecLike.$anonfun$runTests$1(AsyncFlatSpecLike.scala:1767)
	at org.scalatest.AsyncSuperEngine.$anonfun$runTestsInBranch$2(AsyncEngine.scala:437)
	at org.scalatest.Status.$anonfun$thenRun$1(Status.scala:225)
	at org.scalatest.Status.$anonfun$thenRun$1$adapted(Status.scala:223)
	at org.scalatest.ScalaTestStatefulStatus.whenCompleted(Status.scala:607)
	at org.scalatest.Status.thenRun(Status.scala:223)
	at org.scalatest.Status.thenRun$(Status.scala:221)
	at org.scalatest.ScalaTestStatefulStatus.thenRun(Status.scala:503)
	at org.scalatest.AsyncSuperEngine.$anonfun$runTestsInBranch$1(AsyncEngine.scala:437)
	at scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
	at scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
	at scala.collection.immutable.List.foldLeft(List.scala:86)
	at org.scalatest.AsyncSuperEngine.traverseSubNodes$1(AsyncEngine.scala:419)
	at org.scalatest.AsyncSuperEngine.runTestsInBranch(AsyncEngine.scala:475)
	at org.scalatest.AsyncSuperEngine.$anonfun$runTestsInBranch$1(AsyncEngine.scala:460)
	at scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
	at scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
	at scala.collection.immutable.List.foldLeft(List.scala:86)
	at org.scalatest.AsyncSuperEngine.traverseSubNodes$1(AsyncEngine.scala:419)
	at org.scalatest.AsyncSuperEngine.runTestsInBranch(AsyncEngine.scala:483)
	at org.scalatest.AsyncSuperEngine.runTestsImpl(AsyncEngine.scala:550)
	at org.scalatest.AsyncFlatSpecLike.runTests(AsyncFlatSpecLike.scala:1767)
	at org.scalatest.AsyncFlatSpecLike.runTests$(AsyncFlatSpecLike.scala:1766)
	at org.scalatest.AsyncFlatSpec.runTests(AsyncFlatSpec.scala:2219)
	at org.scalatest.Suite.run(Suite.scala:1147)
	at org.scalatest.Suite.run$(Suite.scala:1129)
	at org.scalatest.AsyncFlatSpec.org$scalatest$AsyncFlatSpecLike$$super$run(AsyncFlatSpec.scala:2219)
	at org.scalatest.AsyncFlatSpecLike.$anonfun$run$1(AsyncFlatSpecLike.scala:1812)
	at org.scalatest.AsyncSuperEngine.runImpl(AsyncEngine.scala:620)
	at org.scalatest.AsyncFlatSpecLike.run(AsyncFlatSpecLike.scala:1812)
	at org.scalatest.AsyncFlatSpecLike.run$(AsyncFlatSpecLike.scala:1810)
	at org.scalatest.AsyncFlatSpec.run(AsyncFlatSpec.scala:2219)
	at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:314)
	at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:507)
	at sbt.TestRunner.runTest$1(TestFramework.scala:106)
	at sbt.TestRunner.run(TestFramework.scala:117)
	at sbt.TestFramework$$anon$2$$anonfun$$lessinit$greater$1.$anonfun$apply$1(TestFramework.scala:262)
	at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:233)
	at sbt.TestFramework$$anon$2$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:262)
	at sbt.TestFramework$$anon$2$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:262)
	at sbt.TestFunction.apply(TestFramework.scala:271)
	at sbt.Tests$.$anonfun$toTask$1(Tests.scala:281)
	at sbt.std.Transform$$anon$3.$anonfun$apply$2(System.scala:46)
	at sbt.std.Transform$$anon$4.work(System.scala:66)
	at sbt.Execute.$anonfun$submit$2(Execute.scala:263)
	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
	at sbt.Execute.work(Execute.scala:272)
	at sbt.Execute.$anonfun$submit$1(Execute.scala:263)
	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:174)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: 401:Authentication credentials (https://dev.twitter.com/pages/auth) were missing or incorrect. Ensure that you have set valid consumer key/secret, access token/secret, and the system clock is in sync.
message - Could not authenticate you.
code - 32

Relevant discussions can be found on the Internet at:
	http://www.google.co.jp/search?q=506c3b98 or
	http://www.google.co.jp/search?q=11ed9ae8
TwitterException{exceptionCode=[506c3b98-11ed9ae8], statusCode=401, message=Could not authenticate you., code=32, retryAfter=-1, rateLimitStatus=null, version=4.0.6}
	at twitter4j.HttpClientImpl.handleRequest(HttpClientImpl.java:164)
	at twitter4j.HttpClientBase.request(HttpClientBase.java:57)
	at twitter4j.HttpClientBase.get(HttpClientBase.java:75)
	at twitter4j.TwitterImpl.get(TwitterImpl.java:1786)
	at twitter4j.TwitterImpl.search(TwitterImpl.java:268)
	at com.knoldus.Twitter.retrieveTweet(Twitter.scala:29)
	at com.knoldus.Twitter.$anonfun$getAverageReTweets$1(Twitter.scala:55)
	at scala.runtime.java8.JFunction0$mcI$sp.apply(JFunction0$mcI$sp.java:12)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:654)
	at scala.util.Success.$anonfun$map$1(Try.scala:251)
	at scala.util.Success.map(Try.scala:209)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
</failure>
                                               </testcase>
                     <system-out><![CDATA[]]></system-out>
                     <system-err><![CDATA[]]></system-err>
                   </testsuite>